cs-config/cs_config/functions.py

```diff
@@ -187,7 +188,7 @@ def run_model(meta_param_dict, adjustment):
         OGDIR = os.path.dirname(OGPATH)
         tax_func_path = None#os.path.join(OGDIR, 'data', 'tax_functions',
                         #             cached_pickle)
-        run_micro_baseline = False
+        run_micro_baseline = True
```

Comment:
@jdebacker . It doesn't really matter, but it seems like for cleanliness and consistency that we should move this statement `run_micro_baseline = True` outside of the `if`/`else` statement given that it is `True` in both cases now. Also, I can't figure out what this change does substantially.

Reply:
@rickecon The whole if/else here is not operational since we haven't used the cached pickle file.  By passing `tax_func_path=None`, the model would estimate the tax functions regardless of the boolean being `True` or `False`.  Thus this change has no substantive effect.

I thought we might as well leave in the if/else for now, even if not operational, because I do think we should get something like this working (see Issue #698).  Right now our cached tax functions are DEP, age-specific functions and the C/S app only runs with DEP, non-age-specific, which is why the cached functions aren't used here.

Reply:
@jdebacker OK.